src/components/UserLogin/Login.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import "./Login.css";
+
+const Login = () => {
+	const [email, setEmail] = useState("");
+	const [password, setPassword] = useState("");
+	const [error, setError] = useState("");
+	const [emailError, setEmailError] = useState("");
+	const [showPassword, setShowPassword] = useState(false);
+	const navigate = useNavigate();
+
+	const validateEmail = (email) => {
```

Comment:
I don't think we need to validate an email in the login form. It will either match something in the database or it won't.